.github/workflows/build-and-verify-alignment.yml

```
@@ -0,0 +1,171 @@
+jobs:
+  build-and-verify:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        submodules: recursive
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
```

actionlint (1.7.8)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)